src/components/Pages/People.tsx

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../../types/Person';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader/Loader';
+import { PeopleList } from '../PeopleList/PeopleList';
+
+export const People = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  const isEmpty = people.length === 0;
```

Comment:
Consider using a more descriptive variable name, like 'isPeopleListEmpty' instead of 'isEmpty'